### ISSUE TITLE
Non-unique program error message doesn't go away until refresh

### ISSUE THREAD

**Issue by sanjuroj** [mercycorps/TolaActivity#1409]
When creating a new program, if a non-unique GAIT ID is used, an appropriate error message appears on the field.  However, if you change the GAIT ID and successfully save it, the error message doesn't go away.  It does disappear when the page is refreshed.

**Comment by esinn** [comment]
Does the green successfully appear or not once it's saved? @RickieT

**Comment by RickieT** [comment]
Successful save message appears no matter what I put in the field it seems, but **also does appear** if you try the duplicate, and change it as @sanjuroj wrote in the ticket.
![image](https://user-images.githubusercontent.com/23220180/56247869-1b6e1380-605b-11e9-92b2-0497915d6cda.png)
![image](https://user-images.githubusercontent.com/23220180/56247922-3e002c80-605b-11e9-9425-b10ae1f507c5.png)

**Comment by jennyhavah** [comment]
@axoplasm Do you have time to look at this to let us know how easy/hard it would be to fix? It seems like a fairly edgy edge case, so I don't think it rises to the level of release blocker. But I think it's worth at least spiking on to see how big of an issue it is, before we move it into Deferred.

**Comment by axoplasm** [comment]
Sure thing, I can check on it later this (Weds.) morning.

**Comment by jennyhavah** [comment]
@axoplasm Let's hold off for a moment. We have a slack thread going about eliminating the non-duplication requirement and simply warning you of it instead. Will update this ticket when we've come to a conclusion.

**Comment by jennyhavah** [comment]
After hearing from Cameron, Emily, and Sanjuro that there are no M&E or technical reasons why we must prohibit duplicate GAIT IDs, we've decided to lift this restriction to address valid use cases for programs sharing IDs. I've filed ticket #1429 that replaces this one, and am closing this ticket.

@RickieT heads up that for testing purposes, we no longer need to validate that this ticket passes. We instead need to implement 1429.
